buildpipeline/Core-Setup-Signing-Windows-x86-master.json

```diff
@@ -0,0 +1,491 @@
+{
```

Comment:
Those differences are from the existing MSEng definitions. The x64 and x86 build definitions were created on the same day, and the x64 build was created with the "sign assemblies" step whereas the x86 wasn't, so I assume that that was intentional.

The "BuildProjectsForNuGetPackages" and "GenerateNuGetPackages" targets were added to only the x64 build on 8/4/16 by @livarcocc.

@wtgodbe and @gkhanna79, can you confirm if those differences are intentional?